### ISSUE TITLE
Hi, why I use fairseq0.6.2 to train the translation model does not converge

### ISSUE THREAD

**Issue by jiaohuix** [facebookresearch/fairseq#5070]
### To Reproduce

1. env 

conda create -n seq python=3.6
conda activate seq
conda install pytorch==1.5.0 torchvision==0.6.0 cudatoolkit=10.1 -c pytorch
git clone https://github.com/pytorch/fairseq
cd fairseq
git checkout a8f28ecb63ee01c33ea9f6986102136743d47ec2
pip install --upgrade pip

2. prepare data

cd ./examples/translation/
bash script/prepare-iwslt14.de2en.sh
cd ../../
bash bin.sh

`
DATADIR=data
src=de
tgt=en
TEXT=examples/translation/iwslt14.tokenized.de-en
python preprocess.py --source-lang $src --target-lang $tgt \
  --trainpref $TEXT/train --validpref $TEXT/valid --testpref $TEXT/test \
  --destdir $DATADIR/iwslt14_de_en/  --joined-dictionary \
`

3. train
bash train.sh  data  ckpt
`data=$1
save=$2
mkdir -p $save
fairseq-train    \
    $data  --save-dir $save --max-epoch 50 \
    --arch transformer_iwslt_de_en --share-all-embeddings \
    --optimizer adam --adam-betas '(0.9, 0.98)' --clip-norm 0.0 \
    --lr 5e-4 --min-lr '1e-09'  --lr-scheduler inverse_sqrt --warmup-updates 4000 \
    --dropout 0.3 --weight-decay 0.0001 \
    --criterion label_smoothed_cross_entropy --label-smoothing 0.1 \
    --max-tokens 4096 --tensorboard-logdir $save/vislogs/   >> $save/train.log`

![298739ca334300ef4eed19ade65ab46](https://user-images.githubusercontent.com/68314259/231061758-b466ed00-0928-4fd3-8946-d7b261d64dac.png)

### Environment

 - fairseq Version 0.6.2
 - PyTorch Version   torch1.5
 - OS (e.g., Linux):  linux
 - How you installed fairseq (`pip`, source):   both
 - Build command you used (if compiling from source):
 - Python version: python 3.6
 - CUDA/cuDNN version:  cuda10.1 
 - GPU models and configuration:  2080ti
 - Any other relevant information:
